src/components/auth/SocialAuthButton.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const { REACT_APP_GOOGLE_LOGIN, REACT_APP_FACEBOOK_LOGIN, REACT_APP_TWITTER_LOGIN } = process.env;
+const socialAuthButton = () => (
+  <div className="social-media-Login">
+    <div>
+      <div className="social-auth-text">
+        <span className="valign-wrapper">______________</span>
+          OR LOGIN WITH
+        <span className="valign-wrapper">______________</span>
+      </div>
+    </div>
+    <div className="icon-container">
+      <div>
```

Comment:
Hi Okoro, good job so far! I noticed you were repeating similar code for the facebook, twitter, and google links. This could be made reusable by making it a component and then passing different props to them. So, we could have a <MediaLink /> component that takes in props from facebook, twitter and google. Please let me know if you need help implementing this.

Reply:
resolved.